---
title: "*Sphagnum* Conditioned Microbiome Data and Figures"
author: "Alyssa Carrell"
date: "1/15/2019"
output:
  html_document:
    df_print: paged
    code_folding: hide
    fig_caption: yes
    number_sections: no
    toc: yes
  pdf_document:
    toc: yes
---

# Introduction
The use of microbiomes to select for specific plant host phenotypic response to environment have been demonstrated in recent studies. However, the underlying mechanisms for microbial mediated plant phenotypic response is unknown. Here we explore the mechanisms for microbial mediated host tolerance of elevated temperature. Microbiomes conditioned to ambient or elevated temperature were applied to moss and subjected to ambient or elevated temperatures. Fluorcam measurements were performed each week and the experiment was harvested at 4 weeks to assess plant growth and microbiome composition.

# Experimental details

**Experiment Start Date**	
Experiment 1 - performed 2016  
Experiment 2 - performed 2017
	
**Duration**	
Experiment 1 - 4 weeks  
Experiment 2 - 3 weeks
	
**Measurements taken**	
Experiment 1 - Fluorcam weekly, area growth, 16S profile, RNA not extractable, pH measured  
Experiment 2 - Fluorcam weekly, area growth, 16S profile, RNAseq, final pH, 15N and ARA 
	
**Methods**	
Adapted microbe extraction and inoculation	From each treatment, 100 g of tissue was diced with sterile razor blade and pulverized in BG11 -N, pH 5.5 with a mortar and pestle. The resulting suspension was filtered through Mira Cloth and stored at 4 C until application. The microbes were then pelleted and resuspended in BG11 -N medium (pH5.5). A single capitula of axenic Sphagnum fallax was added to each well of a 12 well plate and inoculated with 2 ml of +0 microbiome, +9 microbiome, or sterile media. Warming conditions were determined from the 2016 summer season temperatures 

# Packages 
```{r warning=FALSE, results='hide', message=FALSE}
library(tidyverse)
library(car)
library(rcompanion)
library(FSA)
library(agricolae)
library(reshape2)
library(plotrix)
```


```{r setup, include=FALSE}

library(captioner)
fig_nums <- captioner(prefix = "Fig.")
fig.1_cap <- fig_nums(name = "fig_1", 
                        caption = "Growth rate of conditioned microbiome 2016")
fig.2_cap <- fig_nums(name = "fig_2", 
                        caption = "Growth rate of conditioned microbiome 2017")
fig.3_cap <- fig_nums(name = "fig_3", 
                        caption = "Maximum quantum yield of Sphagnum with applied microbiomes over duration of experiment 2016")

fig.4_cap <- fig_nums(name = "fig_4", 
                        caption = "Maximum quantum yield of Sphagnum with applied microbiomes over duration of experiment 2017")

```

# Experiment 1 

## Summary Statistics
```{r Experiment 1 Dataset, echo=FALSE, warning=FALSE}
Exp1 <- read.csv("~/Documents/Git/conditioned_microbiome/Microbiome2017/Adapted_Microbiome_Experiment1.csv", header = TRUE, stringsAsFactors = TRUE, na.strings = "NA")

# summary 
Exp1_data_summary <- Exp1 %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    N = length(Growth.rate.mmperday),
    mean = mean(Growth.rate.mmperday),
    sd = sd(Growth.rate.mmperday),
    se = sd/sqrt(N)
  )
```

```{r Figure 1, echo=FALSE, fig.width = 15, fig.height=10, fig.align='center',fig.cap = fig.1_cap}

ggplot(Exp1_data_summary, aes (x=Chamber, y=mean, fill=Microbe)) +
  geom_bar(position=position_dodge(), stat = "identity") +
  geom_errorbar(aes(ymin=mean-se, ymax=mean+se), width=0.2,
                position=position_dodge(0.9)) +
  xlab(" Growth chamber") +ylab("Growth rate (mm/day)") + theme_classic() + theme(axis.text=element_text(size=12),
        axis.text.x = element_text(angle = 0, vjust = 0.5),
        axis.title=element_text(size=16,face="bold"),
        legend.text=element_text(size=16)) + scale_x_discrete(breaks = c("AT", "ET"), labels = c("Ambient Temperature", "Elevated Temperature")) + scale_fill_discrete(labels = c("Ambient Conditioned", "Control", "Elevated Conditioned")) + labs(fill = "Microbiome")
```

```{r Experiment 1 Histogram, echo=FALSE, warning=FALSE}
ggplot(data = Exp1, aes(x = Growth.rate.mmperday)) + geom_histogram(binwidth = 0.5, aes(y = ..density..), colour = "black", fill = "white") + geom_density(alpha=0.2, fill="#FF6666") + ggtitle("Density Histogram of Experiment 1 Growth Rate") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + xlab("Growth rate (mm/day)") + ylab("Density")
```

## Rank Transform
```{r Exp 1. Ranked ANOVA}
exp1.rank <- rank(Exp1$Growth.rate.mmperday)

# binding transform
Exp1 <- cbind(Exp1, exp1.rank)

# two-way ANOVA with ranked data
ranked.exp1.aov <- aov(exp1.rank ~ Chamber + Microbe + Chamber:Microbe , data = Exp1)
Anova(ranked.exp1.aov, type = 3)
TukeyHSD(ranked.exp1.aov)

# compact letter displays 
tx <- with(Exp1, interaction(Chamber, Microbe))
amod <- aov(exp1.rank ~ tx, data = Exp1)
groups <- HSD.test(amod, "tx", group = TRUE)
groups
```

Growth data was rank transformed prior to performing a standard two-way ANOVA. Growth rate in each chamber was dependant on microbial community (p < 0.01). Within the Ambient Temperature (AT) chamber there were no differences in moss growth rate among the three microbial treatments. Growth rate of moss inoculated with an Ambient Temperature Conditioned microbiome (AC) did not differ from the control or moss inoculated with an Elevated Temperature Conditioned microbiome (EC). However, within the Elevated Temperature (ET) chamber, moss growth rate was significantly different between EC and control (p < 0.01). Moss grown with AC and the control had relatively similar growth rates, suggesting that changes in growth rate are not simply due to having a microbiome. Rather, the advantage does appear to be driven by a microbiome pre-adapted to elevated temperatures. 

Between the two chambers, there were no differences in growth rate of AC, Control, and EC. For example, moss grown without a microbiome had a similar growth rate regardless of temperature.

## QY 
```{r Exp 1. Weekly QY Figure}
exp1.qy <- Exp1 %>% 
  select(Chamber, Microbe, QY_max.4, QY_max.3, QY_max.2, QY_max.1, QY_max.0) %>% 
  melt(id = c("Chamber", "Microbe"))

ggplot(data = exp1.qy, aes(x = variable, y = value, fill = Chamber)) + geom_boxplot(colour = "black") + facet_grid(. ~ Microbe) + scale_x_discrete(limits = c("QY_max.0", "QY_max.1", "QY_max.2", "QY_max.3", "QY_max.4"), labels = c("Week 0", "Week 1", "Week 2", "Week 3", "Week 4")) + ylab("Mean QY Max") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + theme(axis.title.x = element_blank())
```

### Week 0
```{r QY_max.0}
wk0.sum <- Exp1 %>% 
  dplyr::select(Sample, Chamber, Microbe, QY_max.0) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.0),
    sd = sd(QY_max.0),
    se = ((sd(QY_max.0))/sqrt(n))
  )

```

### Week 1
```{r QY_max.1}
wk1.sum <- Exp1 %>% 
  select(Sample, Chamber, Microbe, QY_max.1) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.1),
    sd = sd(QY_max.1),
    se = ((sd(QY_max.1))/sqrt(n))
  )
```

### Week 2
```{r QY_max.2}
wk2.sum <- Exp1 %>% 
  select(Sample, Chamber, Microbe, QY_max.2) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.2),
    sd = sd(QY_max.2),
    se = ((sd(QY_max.2))/sqrt(n))
  )
```

### Week 3
```{r QY_max.3}
wk3.sum <- Exp1 %>% 
  select(Sample, Chamber, Microbe, QY_max.3) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.3),
    sd = sd(QY_max.3),
    se = ((sd(QY_max.3))/sqrt(n))
  )
```

### Week 4
```{r QY_max.4}
wk4.sum <- Exp1 %>% 
  select(Sample, Chamber, Microbe, QY_max.4) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.4, na.rm = TRUE),
    sd = sd(QY_max.4, na.rm = TRUE),
    se = ((sd(QY_max.4, na.rm = TRUE))/sqrt(n))
  )
```

In general, there was no significant main effects of the microbial community on QY_Max across the four weeks of Experiment 1.  The temperature of the chamber, not the microbial community, appeared to regulate QY_Max heavily in Weeks 0 and 1, though only until Week 3. As the experiment progressed, cross-over interactions between chamber temperature and microbial communities led to significant interaction terms (p < 0.05), despite no main effect of the microbial community. This suggests that QY_Max likely depends on how microbial communities influence moss within each chamber despite hte lack of a main effect. 

## Mortality
Mortality rate is usually expressed as **death** in a defined **population** during a specified **interval**. In this case, we do not necessarily have individual death per day or per each week. Therefore, percent loss of fluorescence from Week 0 QY will be used as proxy to represent moss death as the treatment continues.

Note: decided to calculate percet of fluorscence left (100% - percent lost) at the end of each week instead of purely percet loss to show a declining graph instead of an increasing graph which could be more confusing to interpret.

```{r Exp 1. Percent Left}
# isolating just the QY data
# calculating percent left (100% - percent loss)
exp1.qy.left <- Exp1 %>% 
  dplyr::select(Chamber, Microbe, QY_max.1, QY_max.2, QY_max.3, QY_max.4, QY_max.0) %>%
  mutate(percent_left1 = (100 - (((QY_max.0 - QY_max.1)/QY_max.0)*100))) %>% 
  mutate(percent_left2 = (100 - (((QY_max.0 - QY_max.2)/QY_max.0)*100))) %>%   
  mutate(percent_left3 = (100 - (((QY_max.0 - QY_max.3)/QY_max.0)*100))) %>%  
  mutate(percent_left4 = (100 - (((QY_max.0 - QY_max.4)/QY_max.0)*100))) 

# summary percent left
# make this into a table
qyleft.summary <- exp1.qy.left %>% 
  select(Chamber, Microbe, percent_left1, percent_left2,percent_left3, percent_left4) %>% 
   group_by(Chamber, Microbe) %>% 
   summarise_all(funs(mean, sd, std.error))

# preparation for plotting
exp1.qy.prep <- exp1.qy.left %>%
  dplyr::select(Chamber, Microbe, percent_left1, percent_left2,percent_left3, percent_left4) %>%
  melt(id = c("Chamber", "Microbe"))


# mean over time
exp1.qy.meanplot <- ggplot(data = exp1.qy.prep, aes(x = variable, y = value,group=1)) + geom_point(stat = 'summary', fun.y=mean) + stat_summary(fun.y = mean, geom = "line") + facet_grid(Microbe ~ Chamber) + scale_x_discrete(limits = c("percent_left1", "percent_left2","percent_left3", "percent_left4"), labels = c("Week 1", "Week 2", "Week 3", "Week 4")) + ylab("Mean QY Max") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + theme(axis.title.x = element_blank()) + scale_y_continuous(limits = c(0, 100))

pdf("Exp1_QYMean.pdf")
plot(exp1.qy.meanplot)
dev.off()

# boxplot over time
exp1.qy.boxplot <- ggplot(data = exp1.qy.prep, aes(x = variable, y = value)) + geom_boxplot() + facet_grid(Microbe ~ Chamber) + scale_x_discrete(limits = c("percent_left1", "percent_left2","percent_left3", "percent_left4"), labels = c("Week 1", "Week 2", "Week 3", "Week 4")) + ylab("QY Max") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + theme(axis.title.x = element_blank())

pdf("Exp1_QYBoxplot.pdf")
plot(exp1.qy.meanplot)
dev.off()

```


```{r Exp 1. Two-way repeated measures MANOVA}
# setting up dataframe
exp1.qy.manova <- exp1.qy.left %>% 
  select(Chamber, Microbe, percent_left1, percent_left2, percent_left3, percent_left4) %>% 
  melt(id = c("Chamber", "Microbe"))
exp1.qy.manova$ID <- seq.int(nrow(exp1.qy.manova))

# rank transforming percentage data
qy1.rank <- rank(exp1.qy.manova$value)
exp1.qy.manova  <- cbind(exp1.qy.manova, qy1.rank)

# establishing factors
exp1.qy.manova <- within(exp1.qy.manova, {
  Chamber <- factor(Chamber)
  Microbe <- factor(Microbe)
  variable <- factor(variable)
})

# means 
exp1.qymanova.sum <- exp1.qy.manova %>% 
  group_by(Chamber, Microbe, variable) %>% 
  summarise(mean = mean(value))

# ANOVA
qy1.left.aov <- with(exp1.qy.manova,
               aov(qy1.rank ~ variable*Chamber*Microbe + Error(ID / (Chamber*Microbe))))
summary(qy1.left.aov)
```

Source code: https://datascienceplus.com/two-way-anova-with-repeated-measures/
https://biostats.w.uib.no/factorial-repeated-measures-anova-two-way-repeated-measures-anova/
https://stats.stackexchange.com/questions/264171/how-to-do-two-way-repeated-measures-anova-using-aov-in-r
validate with: https://www.r-bloggers.com/r-tutorial-series-two-way-repeated-measures-anova/ if time 

# Experiment 2 

## Summary Statistics
```{r Experiment 2 Dataset, echo=FALSE, warning=FALSE}
Exp2 <- read.csv("~/Documents/Git/conditioned_microbiome/Microbiome2017/Adapted_Microbiome_Experiment2.csv", header = TRUE, na.strings = "NA", stringsAsFactors = TRUE)

# summary
Exp2_data_summary <- Exp2 %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    N = length(Growth.rate.mmperday),
    mean = mean(Growth.rate.mmperday, na.rm = TRUE),
    sd = sd(Growth.rate.mmperday, na.rm = TRUE),
    se = sd/sqrt(N)
  )
```

```{r Figure 2, echo=FALSE, fig.width = 15, fig.height=10, fig.align='center',fig.cap = fig.2_cap}
ggplot(Exp2_data_summary, aes (x=Chamber, y=mean, fill=Microbe)) +
  geom_bar(position=position_dodge(), stat = "identity") +
  geom_errorbar(aes(ymin=mean-se, ymax=mean+se), width=0.2,
                position=position_dodge(0.9)) +
  xlab("Chamber") +ylab("Growth rate (mm/day)") + theme_classic() + theme(axis.text=element_text(size=12),
        axis.text.x = element_text(angle = 0, vjust = 0.5),
        axis.title=element_text(size=16,face="bold"),
        legend.text=element_text(size=16)) + scale_x_discrete(breaks = c("AT", "ET"), labels = c("Ambient Temperature", "Elevated Temperature")) + scale_fill_discrete(labels = c("Ambient Conditioned", "Control", "Elevated Conditioned")) + labs(fill = "Microbiome")
```

```{r Experiment 2 Histogram, echo=FALSE, warning=FALSE}
ggplot(data = Exp2, aes(x = Growth.rate.mmperday)) + geom_histogram(binwidth = 0.5, aes(y = ..density..), colour = "black", fill = "white") + geom_density(alpha=0.2, fill="#FF6666") + ggtitle("Density Histogram of Experiment 2 Growth Rate") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + xlab("Growth rate (mm/day)") + ylab("Density")
```


Differences in moss growth rate were based on the influence of microbial treatments in the chambers (p <0.001). Within the ambient temperature chamber (AT), moss grown with AC and moss grown with EC increased growth rate relative to the control (p < 0.05). However, there were no differences in growth between AC and EC suggesting that under ambient temperature, microbial inoculation increases growth irregardless of microbial community diversity. 

Elevated temperatures differentiated the influence of the microbial community on moss growth rate. Moss inoculated with a microbial community conditioned in elevated temperatures had high rates of growth despite elevated temperatures with respect to moss grown with ambient conditioned microbiomes (AC) and the control (p < 0.001). Furthermore, AC moss and control moss did not differ in growth rate suggesting that it is specifically a pre-adapted microbiome that imparts moss with the capacity to tolerate elevated temperatures. 

When comparing microbial treatments across chambers, it is apparent that elevated temperatures severely decreased growth of moss with AC and moss control. Only moss inoculated with EC showed no difference in growth regardless of ambient or elevated temperatures. This indicates that the presence of microbial members capable of surviving in elevated temperatures is generally beneficial, but is especially critical in rising temperatures. 

## Rank Transform
```{r Exp 2 . Ranked ANOVA}
exp2.rank <- rank(Exp2$Growth.rate.mmperday)

# binding transform
Exp2 <- cbind(Exp2, exp2.rank)

# two-way ANOVA with ranked data
ranked.exp2.aov <- aov(exp2.rank ~ Chamber + Microbe + Chamber:Microbe , data = Exp2)
Anova(ranked.exp2.aov, type = 3)
TukeyHSD(ranked.exp2.aov)

# compact letter displays 
tx <- with(Exp2, interaction(Chamber, Microbe))
amod <- aov(exp2.rank ~ tx, data = Exp2)
groups <- HSD.test(amod, "tx", group = TRUE)
groups
```

## QY
```{r Exp 2. Weekly QY Figure}
exp2.qy <- Exp2 %>% 
  select(Chamber, Microbe, QY_max.3, QY_max.2, QY_max.1, QY_max.0) %>% 
  melt(id = c("Chamber", "Microbe"))

ggplot(data = exp2.qy, aes(x = variable, y = value, fill = Chamber)) + geom_boxplot(colour = "black") + facet_grid(. ~ Microbe) + scale_x_discrete(limits = c("QY_max.0", "QY_max.1", "QY_max.2", "QY_max.3"), labels = c("Week 0", "Week 1", "Week 2", "Week 3", "Week 4")) + ylab("Mean QY Max") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + theme(axis.title.x = element_blank())
```

### Week 0
```{r Exp2 QY_max.0, echo=FALSE, warning=FALSE}
wk0.sum2 <- Exp2 %>% 
  select(Sample, Chamber, Microbe, QY_max.0) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.0),
    sd = sd(QY_max.0),
    se = ((sd(QY_max.0))/sqrt(n))
  )
```

### Week 1
```{r Exp2 QY_max.1, echo=FALSE, warning=FALSE}
wk1.sum2 <- Exp2 %>% 
  dplyr::select(Sample, Chamber, Microbe, QY_max.2) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.2),
    sd = sd(QY_max.2),
    se = ((sd(QY_max.2))/sqrt(n))
  )
```

### Week 2
```{r Exp2 QY_max.2, echo=FALSE, warning=FALSE}
wk2.sum2 <- Exp2 %>% 
  dplyr::select(Sample, Chamber, Microbe, QY_max.2) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.2),
    sd = sd(QY_max.2),
    se = ((sd(QY_max.2))/sqrt(n))
  )
```

### Week 3
```{r Exp2 QY_max.3, echo=FALSE, warning=FALSE}
wk3.sum2 <- Exp2 %>% 
  dplyr::select(Sample, Chamber, Microbe, QY_max.3) %>% 
  group_by(Chamber, Microbe) %>% 
  summarise(
    n = n(),
    mean = mean(QY_max.3, na.rm = TRUE),
    sd = sd(QY_max.3, na.rm = TRUE),
    se = ((sd(QY_max.3, na.rm = TRUE))/sqrt(n))
  )
```

## Mortality
Question, why the NAs? Were those dead or just not measured?
```{r Exp 2. Percent Left}
# isolating just the QY data
# calculating percent left (100% - percent loss)
exp2.qy.left <- Exp2 %>% 
  dplyr::select(Sample, Chamber, Microbe, QY_max.1, QY_max.2, QY_max.3, QY_max.0) %>%
  mutate(percent_left1 = (100 - (((QY_max.0 - QY_max.1)/QY_max.0)*100))) %>% 
  mutate(percent_left2 = (100 - (((QY_max.0 - QY_max.2)/QY_max.0)*100))) %>%   
  mutate(percent_left3 = (100 - (((QY_max.0 - QY_max.3)/QY_max.0)*100)))

# summary percent left
# make this into a table
exp2.qyleft.sum <- exp2.qy.left %>% 
  select(Chamber, Microbe, percent_left1, percent_left2,percent_left3) %>% 
   group_by(Chamber, Microbe) %>% 
   summarise_all(funs(mean, sd, std.error))

# prep for plotting percent left
exp2.qy.prep <- exp2.qy.left %>% 
  dplyr::select(Chamber, Microbe, percent_left1, percent_left2, percent_left3)  %>% 
  melt(id = c("Chamber", "Microbe"))

# mean over time
exp2.qymeanplot <- ggplot(data = exp2.qy.prep, aes(x = variable, y = value,group=1)) + geom_point(stat = 'summary', fun.y=mean) + stat_summary(fun.y = mean, geom = "line") + facet_grid(Microbe ~ Chamber) + scale_x_discrete(limits = c("percent_left1", "percent_left2", "percent_left3"), labels = c("Week 1", "Week 2", "Week 3")) + ylab("Mean QY Max") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + theme(axis.title.x = element_blank()) + scale_y_continuous(limits = c(0, 100))

pdf("Exp2_QYMean.pdf")
plot(exp2.qymeanplot)
dev.off()

# boxplot over time
exp2.boxplot <- ggplot(data = exp2.qy.prep, aes(x = variable, y = value)) + geom_boxplot() + facet_grid(Microbe ~ Chamber) + scale_x_discrete(limits = c("percent_left1", "percent_left2", "percent_left3"), labels = c("Week 1", "Week 2", "Week 3")) + ylab("QY Max") + theme_bw() + theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) + theme(axis.title.x = element_blank()) 

pdf("Exp2_Boxplot.pdf")
plot(exp2.boxplot)
dev.off()
```


```{r Exp 2. Two-way repeated measures MANOVA}
# setting up dataframe
exp2.qy.manova <- exp2.qy.left %>% 
  select(Chamber, Microbe, percent_left1, percent_left2, percent_left3) %>% 
  melt(id = c("Chamber", "Microbe"))
exp2.qy.manova$ID <- seq.int(nrow(exp2.qy.left))

# rank transforming percentage data
qy2.rank <- rank(exp2.qy.manova$value)
exp2.qy.manova <- cbind(exp2.qy.manova, qy2.rank)

# establishing factors
exp2.qy.manova <- within(exp2.qy.manova, {
  Chamber <- factor(Chamber)
  Microbe <- factor(Microbe)
  variable <- factor(variable)
})

# means 
exp2.qymanova.sum <- exp2.qy.manova %>% 
  group_by(Chamber, Microbe, variable) %>% 
  summarise(mean = mean(value))

# ANOVA
qy2.left.aov <- with(exp2.qy.manova,
               aov(qy2.rank ~ variable*Chamber*Microbe + Error(ID / (Chamber*Microbe))))
summary(qy2.left.aov)
```

The conditioned microbiome was a primary determinant of QY_Max in Experiment 2. With the exception of the last week (Week 3), the microbiome was the most significant factor in QY_Max. In contrast to Experiment 1, there was only one instance where there appeared to be a strong interaction between the conditioned microbiome and the chamber (Week 2). Typically, it was the microbiome that drove differences in QY_Max between and within both chambers. 

